### PR TITLE
Bug report template: various improvements

### PR DESCRIPTION
**Pull request by jrfnl**
After the umpteenth time of getting a non-reproducible bug report (not just in this repo, mind you), I figured it may help to add some more guidance to the bug report template.

Most important changes:
* Ask for the actual CLI command people have used.
* Ask for the custom ruleset used (if used).
* Give some guidance on how to find out the version numbers of the tooling used.

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -12,13 +17,29 @@ What did you expect to happen? What actually happened?
 -->
 
 ## Minimal Code Snippet
-<!-- Please provide example code that demonstrates the issue. Do NOT paste screenshots of code! -->
+<!-- Please provide example code that allows to reproduce the issue. Do NOT paste screenshots of code! -->
```
Comment (GaryJones) [inline .github/ISSUE_TEMPLATE/bug_report.md:20]:
```suggestion
<!-- Please provide example code that allows anyone to reproduce the issue. Do NOT paste screenshots of code! -->
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -4,6 +4,11 @@ about: Create a report to help us improve
 
 ---
 
+<!--
+PLEASE FILL OUT THE TEMPLATE COMPLETELY.
+ISSUES WHICH CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED WITHOUT HESISTATION.
```
Comment (GaryJones) [inline .github/ISSUE_TEMPLATE/bug_report.md:9]:
```suggestion
ISSUES THAT CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED AS UNACTIONABLE.
```

**Review by GaryJones — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (jrfnl) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: Well, _anyone_ may be expecting a bit much... ?

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 2 above)
Comment (jrfnl) [reply .github/ISSUE_TEMPLATE/bug_report.md:9]: What about simplifying it to:
```diff
+ISSUES WHICH CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED WITHOUT HESISTATION.
-BUG REPORTS WHICH CANNOT BE REPRODUCED BASED ON THE INFORMATION PROVIDED WILL BE CLOSED.
```

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (GaryJones) [reply .github/ISSUE_TEMPLATE/bug_report.md:20]: Feel free to change to "us". :-)

**Comment by jrfnl** [comment]
Updated. I ended up making some additional changes, so added it as a separate commit to allow for easier review of the changes. Please feel free to ask me to squash or to squash on commit.

**Review by GaryJones — APPROVED** [review]

**Comment by GaryJones** [comment]
All yours to merge once the last status check has updated.

**Comment by jrfnl** [comment]
> once the last status check has updated

That reminded me to check whether the "Allow auto-merge once all status checks have passed" option was turned on for this repo. It wasn't, so I have enabled it now ;-)

**Review by dingo-d — APPROVED** [review]

**Pull request merged**